### ISSUE TITLE
Mirrored content formatting enhancement

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/pivotaltrackermirror#3]
Would be great to reduce the focus of the meta-data and provide more focus to the mirrored content.

Taking https://github.com/gberche-orange/pivotal-tracker-mirror/issues/178 as an example, this would translate into:

```
When a user is authenticated to a sample app via an external IdP, and they try to authenticate to another sample app, they get a sad cloud if the second sample app does not have the IdP enabled for the second sample app.

Steps to reproduce:
1. Push the auth code sample app and select an external IdP
1. Push the implicit sample app and do not select an external IdP
1. Log into the auth code sample app via the external IdP
1. Click the log in link for the implicit sample app. (See the screenshot for error) 

Expected result:
1. The user should be redirected to the UAA login page to enter creds in UAA.

---

Mirrors: [story 117497613](https://www.pivotaltracker.com/story/show/117497613) submitted on Apr 13, 2016
- Labels: 
- Requester: Kevin Huang
- Estimate: 0.0
```

while it is currently:

```
- **Link**: https://www.pivotaltracker.com/story/show/117497613
- **Labels**: 
- **Requester**: Kevin Huang
- **Estimate**: 0.0

When a user is authenticated to a sample app via an external IdP, and they try to authenticate to another sample app, they get a sad cloud if the second sample app does not have the IdP enabled for the second sample app.

Steps to reproduce:
1. Push the auth code sample app and select an external IdP
1. Push the implicit sample app and do not select an external IdP
1. Log into the auth code sample app via the external IdP
1. Click the log in link for the implicit sample app. (See the screenshot for error) 

Expected result:
1. The user should be redirected to the UAA login page to enter creds in UAA.
```

Stories with empty descriptions would not display ```null``` as in https://github.com/gberche-orange/pivotal-tracker-mirror/issues/231#issue-167689093

Similarly for comments such as https://github.com/gberche-orange/pivotal-tracker-mirror/issues/178#issuecomment-235334252 this would result in 

```
Madhura Bhave [commented](https://www.pivotaltracker.com/n/projects/997278/stories/119548885/comments/133997707) on May 13, 2016, 9.23pm :

@stummidi pausing this story because we cannot guarantee that the email entered on the first page will only map to one username because emails are not unique.

One alternative is to show the username and password fields one the internal IDP has been "discovered" based on email entered on the first page.

<!--
Comment Id: 134010181
-->
```

while it is currently (note the comment URL typo):

```
- **Link**: https://www.pivotaltracker.com/n/projects/997278/stories/119548885/comments134010181
- **User**: Madhura Bhave

@stummidi pausing this story because we cannot guarantee that the email entered on the first page will only map to one username because emails are not unique.

One alternative is to show the username and password fields one the internal IDP has been "discovered" based on email entered on the first page.

Comment Id: 134010181
```

**Comment by gberche-orange** [comment]
@ArthurHlt in https://github.com/cf-trackermirror-bot/buildpacks-tracker-mirror/issues/36#issuecomment-239445010 the comment URL typo is still there: it links to https://www.pivotaltracker.com/n/projects/1042066/stories/120715555/comments135644775 instead of https://www.pivotaltracker.com/n/projects/1042066/stories/120715555/comments/135644775

**Comment by gberche-orange** [comment]
Fixed typo in 6e63f405c1d336774de0948a971c1235d530dda2

**Comment by gberche-orange** [comment]
Note comment url fix into https://github.com/ArthurHlt/pivotaltracker-java-client/commit/9be49dd2475a9febe9af3247d7510edd019553ca

**Comment by ArthurHlt** [comment]
Mention corrected here: https://github.com/orange-cloudfoundry/pivotaltrackermirror/issues/10

Null description corrected on commit: bce8131
